patchwork/urls.py

```diff
@@ -115,6 +115,11 @@
         name='comment-redirect',
     ),
     # series views
+    path(
+        'project/<project_id>/series/<int:series_id>/',
+        series_views.series_detail,
+        name='series-detail',
+    ),
     path(
```

Comment:
Now that we have a proper series view, could you add a new view for `project/<project_id>/series/<int:series_id>/mbox/` to keep things consistent? That should be a separate patch early in the series and a redirect from the current URL should be provided.